priv/repo/migrations/20161128200526_create_user.exs

```diff
@@ -0,0 +1,12 @@
+defmodule MovieWagerApi.Repo.Migrations.CreateUser do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :username, :string, null: false
```

Comment:
yup yup! So the first part: `create unique_index(:users, [:username])` will be added to the migration. So the migration file will look like this once it's added:
```
defmodule MovieWagerApi.Repo.Migrations.CreateUser do
  use Ecto.Migration

  def change do
    create table(:users) do
      add :username, :string, null: false
      add :profile_image_url, :string

      timestamps()
    end
    
    create unique_index(:users, [:username])
  end
end
```